hammerblade/torch/kernel/hb_tiled_for.hpp

```diff
@@ -76,121 +76,218 @@ inline void calc_range(hb_range* range, size_t numel,
 }
 
 // =========================================================
-// Tiled Pointwise for
+// Tile Pointwise for -- Ternary
 // =========================================================
 
```

Comment:
So ... do we have a conclusion here? What should we do?

Reply:
Do you mean the order of arguments?

Reply:
ahh yes